storage/fuzz/fuzz_targets/adb_any_fixed_sync.rs

```diff
@@ -168,7 +168,7 @@ fn fuzz(input: FuzzInput) {
                 }
 
                 Operation::Commit => {
-                    db.commit().await.expect("Commit should not fail");
+                    db.commit(None).await.expect("Commit should not fail");
```

Comment:
Should we add a fuzz operation to check that the most recently written metadata is correct? We could have the metadata be a counter that tracks the number of commits, or something.

Reply:
done, though I can't run fuzz tests locally for some reason (getting some verbose linker error?). Let's see if it passes on the CI...